WordPress/src/main/java/org/wordpress/android/util/QuickStartUtils.kt

```diff
@@ -96,7 +97,7 @@ class QuickStartUtils {
             )
             // nothing to highlight
             if (startOfHighlight != -1 && endOfHighlight != -1) {
-                val highlightColor = resourceProvider.getColor(android.R.color.white)
+                val highlightColor = resourceProvider.getAttr(activityContext, R.attr.colorSurface)
```

Comment:
And then here we could probably use [this](https://github.com/wordpress-mobile/WordPress-Android/blob/635536d1190c49f40d9a00995440af056e391b13/WordPress/src/main/java/org/wordpress/android/util/ContextExtensions.kt#L22-L27) instead:

```suggestion
                val highlightColor = activityContext.getColorFromAttribute(R.attr.colorSurface)
```